docs/software/remote_desktop.md

```diff
@@ -33,80 +34,212 @@
 
     To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
-## [29] TigerVNC Server - Desktop for remote connection
-
 !!! info Desktop environment
 
     From the list of **Optimised Software**, together with any of the Remote Desktop Software choose also one of [_Graphical Desktop environment_](../desktop). DietPi will install both of them, enabling you to use your keyboard and mouse to interact with a graphical desktop environment on your device.
 
-Run a **Desktop environment** on your device and access it accessed remotely via network. It is a great option for headless SBC devices.
+## [29] TigerVNC Server - Desktop for remote connection
+
+![DietPi remote desktop software TigerVNC](../assets/images/dietpi-software-remote-desktop-tigervnc.jpg){: style="width:600px"}
 
 === "Quick start"
 
-    You can control the VNC service with:
+    You can monitor the VNC service with:
+
     ```
     systemctl status vncserver.service
```

Comment:
Just an info: The `.service` file ending is optional here. `systemctl` adds/assumes it automatically, so `systemctl status vncserver` would work just the same. In case of copy&paste it doesn't matter but saves admins a few keystrokes when dealing regularly with `systemctl` 😉.

Reply:
Ok, then let's get rid of these dull 0.8 seconds. :grin: